clang/lib/CodeGen/CMakeLists.txt

```diff
@@ -31,6 +31,7 @@ set(LLVM_LINK_COMPONENTS
   Target
   TargetParser
   TransformUtils
+  LLVMCodeGenTypes
```

Comment:
These are lexicographically sorted, please maintain that invariant.

Reply:
Oh, didn't realize that.